packages/forklift-console-plugin/src/modules/Providers/views/migrate/index.ts

```diff
@@ -0,0 +1,3 @@
+// @index(['./*', /style/g], f => `export * from '${f.path}';`)
```

Comment:
nice, do use visual studio extension for auto index pages ?

Reply:
no:) I've just copied the markup to maintain consistence across our code base.